tools/vmd/hbonds/hbonds.xml

```diff
@@ -0,0 +1,79 @@
+<tool id="vmd_hbonds" name="Hydrogen Bond Analysis using VMD" version="1.9.3">
+    <description>Analyze H-bonds between two segments</description>
+    <requirements>
+        <requirement type="package" version="1.9.3">vmd</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+<![CDATA[
+     ln -s '$__tool_directory__/hbonds.tcl' ./hbonds.tcl &&
+     vmd -pdb '$strin' -dcd '$trajin' -dispdev text -e '$hbonds_inp' &>> verbose.txt 
+]]></command>
```

Comment:
It would be nice if this could also accept GRO/XTC files as input if possible.

Reply:
I haven't tested GROMACS files, I will and update.

Reply:
Thanks!